### ISSUE TITLE
Can't Reject Incoming Call When Active Connection Present

### ISSUE THREAD

**Issue by neibla** [twilio/client-js-1.4-examples#2]
Scenario:
When on a phone call the client receives an incoming phone call, the client should be able to intercept and reject the incoming call straight away.

Current behaviour:
Incoming call listener is not triggered  and the following log is found 
```
[Device] Device busy; ignoring incoming invite
```

This section seems to be the culprit from http://media.twiliocdn.com/sdk/js/client/releases/1.4.24/twilio.js
```  this.stream.addListener('invite', function(payload) {
    if (self._activeConnection) {
      self.log('Device busy; ignoring incoming invite');
      return;
    }
```

It would be ideal for a setup param to toggle between ignore/reject as a default for this case.
https://www.twilio.com/docs/api/client/device#setup

**Comment by liberty-rowland** [comment]
Thanks for the report @neibla. I'll open a ticket internally for this issue, and respond again here when I have an update.

**Comment by drogit** [comment]
buddy,  did you have any luck getting a response to this issue?  i found (if i understand it correctly) once a call hits twilio.js its done for.  one can either answer the call or reject it (hangup on the caller).  thats it.  the call cannot be enqueued or transferred and like you say, if on a call already, the new call is toast.
as a result of this debacle (a stunning short coming)  i abandoned all plans/code to allow a live call to be answered by twilio.js.  instead, all inbound calls are answered by server code (ivr/auto-attendant) that enqueues the call immediately, sends the agent/user a push notification indicating a caller awaits.  thus the agent clicks that button to answer the call.   of course now that the server has control of  the call, (enqueued)  it can be  handled nicely (hold music, voicemail or forwarded if agent does not pick the call up in a timely manner.  anyway, LMK if you found a way to forward/hold a call within twilio.js.

**Comment by neibla** [comment]
Fixed with 1.6 of the SDK https://www.twilio.com/docs/voice/client/javascript/changelog#160-aug-29-2018
